deployments/zrc/postgres.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres
+# See https://coderjourney.com/using-kubernetes-persistent-volumes/
+
+spec:
+  ports:
+    - port: 5432
+  selector:
+    app: postgres
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: postgres
+spec:
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+        - image: postgre
```

Comment:
postgre, typo ?

versie toevoegen bij image?

Reply:
inderdaad - was even wat wegzetten om morgen te kunnen toetsen met Bart :-) ik laat deze PR voorlopig nog even in WIP